spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -1,12 +1,17 @@
 require "rails_helper"
 
 RSpec.feature "unauthenticated user browses loan requests" do
+  let (:user) { User.create(email: "jorge@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.